aws/v12.1.1/release.diff

```diff
@@ -0,0 +1,64 @@
+    name: coredns                                                      name: coredns
+    version: 1.2.0                                                     version: 1.2.0
+  - componentVersion: 0.7.2                                     |    - componentVersion: 0.7.3
+    name: external-dns                                                 name: external-dns
+    version: 1.2.2                                              |      version: 1.3.0
+  - componentVersion: 3.6.0                                          - componentVersion: 3.6.0
+    name: kiam                                                         name: kiam
+    version: 1.3.1                                              |      version: 1.4.0
+  - componentVersion: 1.9.7                                          - componentVersion: 1.9.7
+    name: kube-state-metrics                                           name: kube-state-metrics
+    version: 1.1.1                                                     version: 1.1.1
+  - componentVersion: 0.3.6                                          - componentVersion: 0.3.6
+    name: metrics-server                                               name: metrics-server
+    version: 1.1.1                                                     version: 1.1.1
+  - name: net-exporter                                               - name: net-exporter
+    version: 1.9.0                                                     version: 1.9.0
+  - componentVersion: 1.0.1                                          - componentVersion: 1.0.1
+    name: node-exporter                                                name: node-exporter
+    version: 1.3.0                                                     version: 1.3.0
+  components:                                                        components:
+  - name: app-operator                                               - name: app-operator
+    version: 1.0.0                                                     version: 1.0.0
+  - name: aws-cni                                                    - name: aws-cni
+    version: 1.6.3                                                     version: 1.6.3
+  - name: aws-operator                                               - name: aws-operator
+    releaseOperatorDeploy: true                                        releaseOperatorDeploy: true
+    version: 8.7.5                                              |      version: 8.8.0
+  - name: calico                                                     - name: calico
+    version: 3.15.1                                                    version: 3.15.1
+  - name: cert-operator                                              - name: cert-operator
+    releaseOperatorDeploy: true                                        releaseOperatorDeploy: true
+    version: 0.1.0                                                     version: 0.1.0
+    reference: 0.1.0-2                                          <
```

Comment:
I've updated the diff, and this change showed up. Is cert-operator reference intentionally removed in 12.1.1, or was it introduced retroactively in 12.1.0? If latter, should same or some different cert-operator reference be used in 12.1.1 as well?

Maybe @giantswarm/team-ludacris as cert-operator owners can help answer this one.

Reply:
It must be `0.1.0-2` in all releases so please update it in `release.yaml` manually for now. I'm checking the latest `devctl` and it uses [apiextensions@0.4.19](https://github.com/giantswarm/apiextensions/blob/v0.4.19/pkg/apis/release/v1alpha1/release_types.go#L107) which includes this field so it should be preserved when reading, editing, and writing the new release. I'm not sure why it is being lost. I will try to investigate later today or tomorrow.

Reply:
Done, thanks!